api/server.py

```diff
@@ -199,13 +201,15 @@ def receive_rated_metapaths():
     'metapath': ['Phenotype', 'HAS', 'Association', 'HAS', 'SNP', 'HAS', 'Phenotype'],
     'rating': 0.75}
     """
+    time_results_received = datetime.datetime.now()
```

Comment:
Why don't we immediately store it in the session variable here?

Reply:
Because we want to save it along with the batch of rated meta-paths but the checking for consistency of the batch is not before the next line